### PR TITLE
[TO BE MERGED 6/30] BD-555

### PR DESCRIPTION
**Pull request by timothy-kim**
BD-555: Add information about variant retention reports to the canvas and campaign retention report articles

# Pull Request/Issue Resolution

**Description of Change:**
> I'm changing..... (could be a link, a new image, a new section, etc.)... because...

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_user_guide/engagement_tools/campaigns/testing_and_more/retention_reports.md [hunk 1]
```diff
@@ -85,6 +87,24 @@ This information can be useful if you want to know the percentage of users who h
...
+- See what retention looks like if you didn’t send that message by digging into the Control group’s retention data.<br><br>
+
+![View by Variant][8]
+
+__Braze Retention Reports By Variant Components:__
+- __Date Range__: Set on the Campaign/Canvas Details page, the date range includes all users who received the campaign during this window, and of those users, the data of those that performed their retention event during the date range will appear in the report. Each day the retention rate, percentage change from the control group, and confidence are measured.
+- __Retention Rate__: Shows the retention rate by variant. The retention rate is equivalent to the number of users that performed the retention event divided by the total users that have received the campaign/canvas.
+- __Recentage Change from Control__: Quantifies the percentage change per variant from the control group.
+- __Confidence__: Braze compares each variant’s conversion rate against the control’s conversion rate with a statistical procedure called a Z Test to calculate a [confidence][3] percentage. This percentage signifies how confidently that variant is performing better than the control group.
```
Comment (KellieHawks) [inline _docs/_user_guide/engagement_tools/campaigns/testing_and_more/retention_reports.md:104]: Broken link here: Confidence[3]

File: _docs/_user_guide/engagement_tools/canvas/retention_reports.md [hunk 2]
```diff
@@ -84,6 +84,25 @@ This information can be useful if you want to know the percentage of users who h
...
+- See what retention looks like if you didn’t send that message by digging into the Control group’s retention data.<br><br>
+
+![View by Variant][8]
+
+__Braze Retention Reports By Variant Components:__
+- __Date Range__: Set on the Campaign/Canvas Details page, the date range includes all users who received the campaign during this window, and of those users, the data of those that performed their retention event during the date range will appear in the report. Each day the retention rate, percentage change from the control group, and confidence are measured.
+- __Retention Rate__: Shows the retention rate by variant. The retention rate is equivalent to the number of users that performed the retention event divided by the total users that have received the campaign/canvas.
+- __Recentage Change from Control__: Quantifies the percentage change per variant from the control group.
+- __Confidence__: Braze compares each variant’s conversion rate against the control’s conversion rate with a statistical procedure called a Z Test to calculate a [confidence][3] percentage. This percentage signifies how confidently that variant is performing better than the control group.
```
Comment (KellieHawks) [inline _docs/_user_guide/engagement_tools/canvas/retention_reports.md:101]: Same broken link.

File: _docs/_user_guide/engagement_tools/campaigns/testing_and_more/retention_reports.md (hunk 1 above)
Comment (timothy-kim) [reply _docs/_user_guide/engagement_tools/campaigns/testing_and_more/retention_reports.md:104]: fixed!

File: _docs/_user_guide/engagement_tools/canvas/retention_reports.md (hunk 2 above)
Comment (timothy-kim) [reply _docs/_user_guide/engagement_tools/canvas/retention_reports.md]: fixed!

**Pull request merged**